src/pages/office/office.ts

```diff
@@ -596,6 +596,9 @@ export class OfficePage extends PracticeableBasePageComponent {
   }
 
   candidateDescriptionChanged(candidateDescription: string) {
+    if (candidateDescription.length > 1000) {
```

Comment:
Perhaps it would be better to prevent the presentational components from emitting when they are invalid instead of having to guard in the container component?

Reply:
Good idea, will make the change